File: backjoon/4153.cpp
#include <iostream>
using namespace std;

int swap(int *a, int *b)
{
	int temp = *a;
	*a = *b;
	*b = temp;
}

int main(void)
{
	int a, b, c;
	while (1)
	{
		cin >> a >> b >> c;
		if (a > b)
			swap(a, b);
		if (b > c)
			swap(b, c);
		if (a == 0)
			return 0;
		else if (c * c == a * a + b * b)
			cout << "right" << '\n';
		else
			cout << "wrong" << '\n';
	}
}

File: backjoon/4153.py
while True:
    nums = list(map(int, input().split()))
    nums.sort()
    if max(nums) == 0:
        exit(0)
    if nums[2] * nums[2] == (nums[0] * nums[0]) + (nums[1] * nums[1]):
        print('right')
    else:
        print('wrong')